### ISSUE TITLE
`ameba --gen-config` yields a .ameba.yml with no rules

### ISSUE THREAD

**Issue by szTheory** [crystal-ameba/ameba#115]
When I run `ameba --gen-config` it just gives me an .ameba.yml with a comment at the top. 

```
# This configuration file was generated by `ameba --gen-config`
# on 2019-09-17 16:55:16 UTC using Ameba version 0.10.1.
# The point is for the user to remove these configuration records
# one by one as the reported problems are removed from the code base.
```

Also, how do I disable specific rules? I want to disable this one, but can't find docs for the list of rules to disable.

```
spec/test_zaru.cr:1:9
[E] Lint/Syntax: unterminated char literal, use double quotes for strings
> require 'spec'
```

I tried adding this to .ameba.yml but it `ameba` still reports the rule, so it's not taking

```
Lint/Syntax:
    Description: unterminated char literal, use double quotes for strings
    Enabled: false
```

Thanks for reading!

**Comment by veelenga** [comment]
Ameba says you have a syntax error in your code. This particular rule cannot be disabled. Update your code:

```diff
- require 'spec'
+ require "spec"
```

**Comment by veelenga** [comment]
Fixed by 2704a0d8b1fd2d62c221030349b56dc82d5691a0
